src/App.js

```diff
@@ -8,7 +14,24 @@ const App = () => {
       <SideBarWrap>
         <SideBar />
       </SideBarWrap>
-      <ModuleSelectionPage />
+      <ContentWrap>
+        {/* Routes for NeoCirc educational platform */}
+        <Routes>
+          {/* General Pages */}
+          <Route path="/" element={<HomePage />} />
+          <Route path="/modules" element={<ModuleSelectionPage />} />
+
+          {/* Content 1 routes */}
+          <Route path="/modules/content1/1" element={<ModuleContent1 />} />
```

Comment:
i feel '/1' is redundant.

Reply:
I was thinking of that as different pages of content1 (ex: /content1/1 for page 1, /content1/2 for page 2), but after thinking about it, it'll probably be easier if we handle it as states instead of different routes.

Thanks for the comment!

Reply:
Fixed